src/dvclive/lightning.py

```diff
@@ -20,10 +20,9 @@ def __init__(
 
         super().__init__()
         self._prefix = prefix
-        self._live_init = {
-            "dir": dir,
-            "resume": resume,
-        }
+        self._live_init = {"resume": resume}
```

Comment:
```suggestion
        self._live_init: Dict[str, Any] = {"resume": resume}
```

Reply:
yep, fixed that already (mypy didn't run for some reason locally)

Reply:
You need to use `nox -s lint` to run all of the linters. See https://github.com/iterative/dvclive/blob/main/CONTRIBUTING.rst#how-to-submit-changes.